src/style.css

```diff
@@ -2,0 +1,46 @@
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 0;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here you have set the bottom margin to 0 for the `.stars` class, which is fine. Just ensure that you're not adding a top margin to this class anywhere else to keep the consistency.